### PR TITLE
Correcting link

### PR DESCRIPTION
**Pull request by get-itips**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mail-flow-best-practices/manage-accepted-domains/manage-accepted-domains.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ There are two types of accepted domains in Exchange Online:
 
     - **Exchange Online**: [Manage mail users](../../recipients-in-exchange-online/manage-mail-users.md)
 
-    - **Exchange Online Protection**: [Manage Mail Users in EOP](https://docs.microsoft.com/microsoft-365/security/office-365-security/manage-mail-users-in-eopx)
+    - **Exchange Online Protection**: [Manage Mail Users in EOP](https://docs.microsoft.com/microsoft-365/security/office-365-security/manage-mail-users-in-eop)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/manage-accepted-domains/manage-accepted-domains.md:37]:
```suggestion
    - **Exchange Online Protection**: [Manage mail users in EOP](https://docs.microsoft.com/microsoft-365/security/office-365-security/manage-mail-users-in-eop)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - some additional minor edits for consistency, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**